packages/main-nav/src/MainNav.tsx

```diff
@@ -2,7 +2,7 @@ import { NavContainer, NavContainerProps } from './NavContainer';
 import { NavList, NavListProps } from './NavList';
 
 export function MainNav({
-	variant = 'darkAlt',
+	variant = 'agriculture',
```

Comment:
I don't think we should set the default variant to `agriculture` for any package